src/Examples/Flowsheets/power_generation/rsofc/cpu.py

```diff
@@ -0,0 +1,720 @@
+##############################################################################
```

Comment:
This needs to have the IDAES copyright header (in addition to the ARPA-E one).

On a related note, I see that the property package code does not have the ARPA-E header (but does have the IDAES one) - that strikes me as very odd.

Reply:
I will follow up with @AlexNoring regarding what files require which headers.

Reply:
The only files that are copied from the ARPA-E project and require headers are cpu.py and cpu_surrogate_methods.py.

Reply:
Thank you. I will add the appropriate headers.